spec/unit/provider/package/windows/exe_package_spec.rb

```diff
@@ -34,7 +34,6 @@
     {
       'DisplayName'      => ['My App', ''],
       'UninstallString'  => ['E:\uninstall.exe', ''],
-      'SystemComponent'  => [nil, 1],
       'WindowsInstaller' => [nil, 1],
       'ParentKeyName'    => [nil, 'Uber Product'],
       'Security Update'  => [nil, 'KB890830'],
```

Comment:
I'm not sure about Security Updates or Hotfixes for that matter, e.g. a user might try to `ensure => absent` and I don't know that that would work in all cases.